source/docs/software/actuators/pneumatics.rst

```diff
@@ -112,3 +112,38 @@ Double solenoids are controlled by the DoubleSolenoid class in WPILib. These are
         exampleDouble.Set(frc::DoubleSolenoid::Value::kOff);
         exampleDouble.Set(frc::DoubleSolenoid::Value::kForward);
         exampleDouble.Set(frc::DoubleSolenoid::Value::kReverse);
+
+
+Pressure Transducers
+--------------------
+
+One can connect a pressure transducer to measure the pressure stored in a pneumatic system. These transducers connect to the Analog Input ports on the roboRIO, and can be read by the AnalogInput or AnalogPotentiometer classes in WPILib.
+
+.. tabs::
+
+    .. code-tab:: java
+
+        import edu.wpi.first.wpilibj.AnalogInput;
+        import edu.wpi.first.wpilibj.AnalogPotentiometer;
+
+        // product-specific voltage->pressure conversion, see product manual
+        // in this case, 250(V/5)-25
+        // the scale parameter in the AnalogPotentiometer constructor is scaled from 1 instead of 5,
+        // so if r is the raw AnalogPotentiometer output, the pressure is 250r-25
+        double scale = 250, offset = -25;
+        AnalogPotentiometer pressureTransducer = new AnalogPotentiometer(/* the AnalogIn port*/ 2, scale, offset);
+
+        // scaled values in psi units
+        double psi = pressureTransducer.get();
+
+    .. code-tab:: c++
+
+        // product-specific voltage->pressure conversion, see product manual
+        // in this case, 250(V/5)-25
+        // the scale parameter in the AnalogPotentiometer constructor is scaled from 1 instead of 5,
+        // so if r is the raw AnalogPotentiometer output, the pressure is 250r-25
+        double scale = 250, offset = -25;
+        frc::AnalogPotentiometer pressureTransducer {/* the AnalogIn port*/ 2, scale, offset};
```

Comment:
```suggestion
        frc::AnalogPotentiometer pressureTransducer{/* the AnalogIn port*/ 2, scale, offset};
```

Reply:
@Daltz333 - what do you say about this change? All other cpp code is with 
```cpp
namespace::Type variableName {args}
```
format. 
If we want to remove the space before the arg braces, it should be done for all code on the page or even all code in frc-docs.

Reply:
That is correct. Code should follow the WPILib style guide.

Reply:
And wpiformat has no space before args?